src/Ragel.jl

```diff
@@ -229,18 +229,18 @@ end
 # Open functions for various sources
 # ----------------------------------
 
-function Base.open{T<:FileFormat}(filename::AbstractString, ::Type{T}; args...)
+function Base.open{T<:FileFormat}(filename::AbstractString, ::Type{T}, args...; kwargs...)
     memory_map = false
     i = 0
-    for arg in args
+    for arg in kwargs
         i += 1
         if arg[1] == :memory_map
```

Comment:
Ah I see each iteration `arg` looks like a tuple. I follow you now. Seems mornings are not good to me reading code.